scripts/installscripts/stretch-install-spotify.sh

```diff
@@ -463,8 +463,10 @@ sudo iwconfig wlan0 power off
 
 # Install required packages
 sudo apt-get update
-sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
+sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python3-dev python-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python-mutagen python3-gpiozero
```

Comment:
```suggestion
sudo apt-get --yes --force-yes install apt-transport-https samba samba-common-bin python3-dev python3-pip gcc linux-headers-4.9 lighttpd php7.0-common php7.0-cgi php7.0 php7.0-fpm at mpd mpc mpg123 git ffmpeg python3-mutagen python3-gpiozero
```

Reply:
We need to keep in mind that on stretch mopidy is available only at version 2.2.3 at the most with `python:any (<< 2.8), python:any (>= 2.7.5-5~)` as python version limits.